modules/apps/client-extension/client-extension-type-impl/build.gradle

```diff
@@ -15,4 +15,6 @@ dependencies {
 	compileOnly project(":core:petra:petra-function")
 	compileOnly project(":core:petra:petra-reflect")
 	compileOnly project(":core:petra:petra-string")
+
+	testImplementation group: "org.osgi", name: "osgi.core", version: "6.0.0"
```

Comment:
@thiago-buarqque This is causing stable to fail. Do we need this dependency added? We already have osgi.core added at line 8. https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/565076/testReport/com.liferay.portal.modules/ModulesStructureTest/testScanGradleFiles/

Reply:
Ops, missed that one.